migrations/automatic/educandu-2021-12-20-01-create-rooms-collection.js

```diff
@@ -0,0 +1,40 @@
+// eslint-disable-next-line camelcase
+export default class Educandu_2021_12_20_01_create_rooms_colletion {
+  constructor(db) {
+    this.db = db;
+  }
+
+  async up() {
+    await this.db.createCollection('rooms');
+    await this.db.createCollection('roomInvitations');
+
+    await this.db.collection('rooms').createIndexes([
+      {
+        name: '_idx_owner_',
+        key: { owner: 1 }
+      },
+      {
+        name: '_idx_members_user_id',
```

Comment:
For consistency we need another underscore at the end

Reply:
Fixed in both files. Thanks!